### PR TITLE
docs: write docs on new scn

### PR DESCRIPTION
**Pull request by NikitaZotov**

### CODE REVIEW

File: docs/sc_web.tex [hunk 1]
```diff
@@ -1,60 +1,74 @@
...
-\scnexplanation{\textit{Панель визуализации и редактирования знаний} отображает окна, содержащие sc-текст, представленный на некотором языке из множества \textit{внешних языков визуализации SC-кода} и, как правило, являющийся результатом некоторого действия, инициированного пользователем. Если соответствующий визуализатор поддерживает возможность редактирования текстов соответствующего естественного языка, то он одновременно является также и редактором.}
-\scnrelfromlist{компонент программной системы}{Визуализатор sc.n-текстов;Визуализатор и редактор sc.g-текстов}
-\scnnote{При необходимости пользовательский интерфейс каждой конкретной ostis-системы может быть дополнен визуализаторами и редакторами различных внешних языков, которые в текущей версии \textit{Реализации интерпретатора sc-моделей пользовательских интерфейсов} будут также располагаться на \textit{Панели визуализации и редактирования знаний}.}
-\scnaddlevel{-1}}
+\begin{scnrelfromlist}{используемый язык программирования}
+    \scnitem{JavaScript}
+    \scnitem{TypeScript}
+    \scnitem{Python}
+\end{scnrelfromlist}
+\scnrelfrom{иллюстрация}{\scnfileimage{images/sc_web_new_arch}}
```
Comment (VasKho) [inline docs/sc_web.tex:9]:
```suggestion
\scnrelfrom{иллюстрация}{\scnfileimage{images/sc\_web\_new\_arch}}
```

**Review by VasKho — CHANGES_REQUESTED** [review]
We don't have following commands in `scn-latex-plugin`
```TeX
\scnexplanation
\scnnote
\begin{scnsubdividing} % i.e. environment
```
If you're really would like to use you should define this commands in main file or simply replace them with `\scnrelfrom{пояснение}`, `\scnrelfrom{примечание}`, `\begin{scnrelfromset}{разбиение}` respectively.

**Review by VasKho — CHANGES_REQUESTED** [review]

File: docs/sc_web.tex [hunk 2]
```diff
@@ -1,60 +1,74 @@
...
+    \scnfileitem{Протокол HTTP предполагает четкое разделение активного клиента и пассивного сервера, который отвечает на запросы клиентов. Таким образом, сервер (в данном случае -- sc-память) практически не имеет возможности по своей инициативе отправить сообщение клиенту, что повышает безопасность системы, но значительно снижает ее интерактивность. Кроме того, такой вариант реализации затрудняет реализацию принятого в Технологии OSTIS многоагентного подхода, в частности, затрудняет реализацию sc-агентов на стороне клиента. Указанные проблемы могут быть решены путем постоянного мониторинга определенных событий со стороны клиента, однако такой вариант неэффективен.
+    Кроме того, часть интерфейса фактически работает напрямую с sc-памятью с использованием технологии WebSocket, а часть -- через прослойку на базе библиотеки tornado для языка программирования Python, что приводит к дополнительным зависимостям от сторонних библиотек.};
```
Comment (VasKho) [inline docs/sc_web.tex:16]:
```suggestion
    Кроме того, часть интерфейса фактически работает напрямую с sc-памятью с использованием технологии WebSocket, а часть -- через прослойку на базе библиотеки tornado для языка программирования Python, что приводит к дополнительным зависимостям от сторонних библиотек.}
```

File: docs/sc_web.tex [hunk 3]
```diff
@@ -1,60 +1,74 @@
...
+    \scnfileitem{Идея платформенной независимости пользовательского интерфейса (построения sc-модели пользовательского интерфейса) реализована не в полной мере. Полностью описать sc-модель пользовательского интерфейса (включая точное размещение, размеры, дизайн компонентов, их поведение и др.) в настоящее время скорее всего окажется затруднительно из-за ограничений производительности, однако вполне возможно реализовать возможность задания вопросов ко всем компонентам интерфейса, изменить их расположение и т.д., однако эти возможности нельзя реализовать в текущей версии реализации платформы.}
+    \scnfileitem{Интерфейсная часть работает медленно из-за некоторых недостатков реализации серверной части на языке Python.}
+    \scnfileitem{Не реализован механизм наследования при добавлении новых внешних языков. Например, добавление нового языка даже очень близкого к SCg-коду требует физического копирования кода компонента и внесение соответствующих изменений, при этом получаются два никак не связанных между собой компонента, которые начинают развиваться независимо друг от друга.};
```
Comment (VasKho) [inline docs/sc_web.tex:23]:
```suggestion
    \scnfileitem{Не реализован механизм наследования при добавлении новых внешних языков. Например, добавление нового языка даже очень близкого к SCg-коду требует физического копирования кода компонента и внесение соответствующих изменений, при этом получаются два никак не связанных между собой компонента, которые начинают развиваться независимо друг от друга.}
```

**Review by VasKho — APPROVED** [review]

**Review by KovalM — APPROVED** [review]

**Pull request merged**